bitnami/cassandra/values.yaml

```diff
@@ -164,6 +164,11 @@ cluster:
   rack: rack1
   enableRPC: true
   endpointSnitch: SimpleSnitch
+  ## Enable the creation of the Pod Disruption Budget
+  ##
+  pdbEnabled: false
```

Comment:
In order to not make it a major change, I would set it to true by default so it does not change the expected behavior between versions.